pkgs/top-level/python-packages.nix

```diff
@@ -4610,6 +4610,10 @@ in {
 
   seqdiag = callPackage ../development/python-modules/seqdiag { };
 
+  sequoia = callPackage ../tools/security/sequoia {
```

Comment:
toPythonModule ( callPackage ....

Reply:
Thanks for the thorough review. I think I've fixed all of the issues, as for this one, I'm less sure.

As for the rest of the issues, the build takes about half an hour or so and it's very CPU intensive for my computer :( Please excuse me for taking the risk and not building it all the way to the end but I'm quiet certain it should work because I checked the environment in a `nix-shell` and the important environment variables `PYTHONPATH` and `LIBCLANG_PATH` have good values.